packages/webview_flutter/example/android/app/build.gradle

```diff
@@ -56,7 +56,8 @@ flutter {
 
 dependencies {
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.2.0'
-    androidTestImplementation 'androidx.test:rules:1.2.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    testImplementation "com.google.truth:truth:1.0"
+    androidTestImplementation 'androidx.test:runner:1.1.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
```

Comment:
Why are we downgrading several dependencies?

Reply:
I copied the versions from the `espresso` plugin because I thought it might be related to an error. I converted them back.